### PR TITLE
Add optional "lookupHeader" to replace default header

### PR DESCRIPTION
**Pull request by Dominic-Preap**
Update for issue #179.

### CODE REVIEW

File: src/languageLookups/header.js [hunk 1]
```diff
@@ -8,8 +8,11 @@ export default {
       let headers = req.headers;
       if (!headers) return found;
 
+      let lookupExist = options.lookupHeader !== undefined; 
+      let lookupHeader = lookupExist ? headers[options.lookupHeader] : undefined;
+
       let locales = [];
-      let acceptLanguage = headers['accept-language'];
+      let acceptLanguage = lookupHeader || headers['accept-language'];
```
[the remark this answers is not in the document]
Comment (Dominic-Preap) [reply src/languageLookups/header.js]: As first, I thought I want to have a fallback to default if `lookupHeader` is provided but empty. It seems I don't have to do that.

**Comment by Dominic-Preap** [comment]
@jamuhl Done, I've just changed to one liner.

**Comment by jamuhl** [comment]
published in i18next-express-middleware@1.8.0

**Comment by jamuhl** [comment]
If you like this module don’t forget to star this repo. Make a tweet, share the word or have a look at our https://locize.com to support the devs of this project -> there are many ways to help this project :pray:

**Pull request merged**